lib/kitchen/provisioner/salt_solo.rb

```diff
@@ -274,7 +274,7 @@ def salt_command
         end
 
         if config[:pre_salt_command]
-          cmd << "#{config[:pre_salt_command]} && "
+          cmd << "( set -x ; #{config[:pre_salt_command]} ) && "
```

Comment:
Actually, `set` is linux specific, what happens if we're running on a windows VM?